ietf/templates/group/meetings-row.html

```diff
@@ -57,14 +57,24 @@
                 Minutes
             </a>
             <a class="btn btn-primary btn-sm"
-               href="{% url 'ietf.meeting.views.session_details' num=s.meeting.number acronym=s.group.acronym %}">
+                href="{% url 'ietf.meeting.views.session_details' num=s.meeting.number acronym=s.group.acronym %}">
+                {% if can_always_edit or can_edit_materials %}
+                    <i class="bi bi-pencil-square">
+                    </i>
+                {% endif %}
                 Materials
             </a>
-            {% if can_always_edit or can_edit_materials %}
-                <a class="btn btn-info btn-sm float-end"
-                   href="{% url 'ietf.meeting.views.session_details' num=s.meeting.number acronym=s.group.acronym %}">
-                    Edit Materials
-                </a>
+            {% if not s.cancelled %}
+                <div class="regular float-end">
+                    {# see note in the included templates re: show_agenda parameter and required JS import #}
+                    {% if s.meeting.type.slug == 'interim' %}
+                        {% include "meeting/interim_session_buttons.html" with show_agenda=False show_empty=False session=s meeting=s.meeting use_notes=s.meeting.use_notes %}
+                    {% else %}
+                        {% with schedule=s.meeting.schedule %}
```

Comment:
I don't see anything using this `schedule` variable? Drop? (Or make a parameter to the `{% include %}` instead, if somehow it is being used?)